test/e2e/nftables_test.go

```diff
@@ -23,15 +34,21 @@ var (
 	chainName      = "chain OPENSHIFT"
 )
 
+const (
+	waitDuration = 2 * time.Minute
+	mcTimeout    = 20 * time.Minute
+)
+
 var _ = Describe("Nftables", func() {
 	It("should apply firewall by blocking all ports except the ones OCP is listening on", func() {
 		masterMat, workerMat := commatrix.SeparateMatrixByRole()
 		var workerNFT []byte
-
+		nodeRoles := []string{"master"}
```

Comment:
Why don't you get a list of all the roles instead of hardcoding the roles to `master` and `worker`?
Cluster nodes may have other roles (e.g. `infra`, `worker-sriov`, etc.).